scripts/test-runner.js

```diff
@@ -201,15 +217,26 @@ process.on('SIGINT', shutdownProcesses);
 process.on('SIGTERM', shutdownProcesses);
 process.on('SIGHUP', shutdownProcesses);
 
-startRC().then(() => {
+startRC().then(async () => {
     console.log('Hazelcast Remote Controller is started!');
     if (runTests) {
-        console.log(`Running tests... Test type: ${testType}, Test command: ${testCommand}`);
+        console.log(`Running ${testType}, Command: ${testCommand}`);
         testProcess = spawn(testCommand, [], {
             stdio: ['ignore', 'inherit', 'inherit'],
             shell: true
         });
         testProcess.on('exit', shutdownRC);
+    } else if (testType === 'check-code-samples') {
+        const RC = getRC();
+        cluster = await RC.createClusterKeepClusterName(null, TestUtil.createClusterConfig({clusterName: 'dev'}));
```

Comment:
I think you should be able to pass `null` for the cluster config. After all, the default value for the clusterName is `dev`, so the default config should also be enough for us.

Then you can un-export the createClusterConfig.

Reply:
default value is not dev, it is random uuid. check out https://github.com/hazelcast/hazelcast-remote-controller/blob/master/src/main/java/com/hazelcast/remotecontroller/HzCluster.java

Reply:
Then I would just pass the config as a string. I think using the test utilities here is not a good practice.
```
<?xml version="1.0" encoding="UTF-8"?>
<hazelcast xmlns="http://www.hazelcast.com/schema/config"
    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
    xsi:schemaLocation="http://www.hazelcast.com/schema/config
    http://www.hazelcast.com/schema/config/hazelcast-config-4.0.xsd">
    <cluster-name>dev</cluster-name>
</hazelcast>
```